src/runtime_src/core/pcie/driver/linux/xocl/subdev/ert_user.c

```diff
@@ -1079,21 +1073,35 @@ static int ert_cfg_cmd(struct xocl_ert_user *ert_user, struct ert_user_command *
 		ERTUSER_ERR(ert_user, "should not have zeroed value of cq_size=%lld, slot_size=%d",
 		    ert_user->cq_range, cfg->slot_size);
 		return -EINVAL;
+	} else if (cfg->slot_size & 0x1F) {
+		ERTUSER_ERR(ert_user, "slot_size should be 32-bit aligned, slot_size=%d",
+		   cfg->slot_size);
+		return -EINVAL;
 	}
```

Comment:
The slot_size is in bytes. "cfg->slot_size & 0x1F" here means it is 32 bytes aligned. Right?